R/applications.R

```diff
@@ -255,77 +254,66 @@ showLogs <- function(appPath = getwd(), appFile = NULL, appName = NULL,
   }
 }
 
-#' Sync Application Metadata
+#' Update deployment records
 #'
-#' Update the metadata for requested application across all deployments
+#' Update the deployment records for applications published to Posit Connect.
+#' Updates application title and URL, and deletes records for deployments
+#' where the application has been deleted on the server.
 #'
 #' @param appPath The path to the directory or file that was deployed.
-#'
-#' @note This function does not update metadata for Shiny and rpubs apps
-#'
+#' @param cacheHours Number of hours to wait between confirming
```

Comment:
Adding this argument made it easier to test. But given how little it takes to run and that `rsconnect.metadata.sync.hours` doesn't appear to be used anywhere else, I think it makes sense to remove the `lastSyncTime` component all together.

Reply:
I don't know of any calls from the IDE to this function; I feel like we could remove all notion of "time" (no skipped updates, no update timestamps).